frontend/app/src/main/java/com/intersoft/groupup_app/navigation/EditEventPage.kt

```diff
@@ -0,0 +1,141 @@
+package com.intersoft.groupup_app.navigation
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableLongStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.intersoft.auth.AuthContext
+import com.intersoft.event.EventManager
+import com.intersoft.ui.CounterElement
+import com.intersoft.ui.DurationSelectionElement
+import com.intersoft.ui.ErrorText
+import com.intersoft.ui.GeneralDatePicker
+import com.intersoft.ui.MultiLineTextInputField
+import com.intersoft.ui.PrimaryButton
+import com.intersoft.ui.TextInputField
+import com.intersoft.ui.TitleText
+import com.intersoft.ui.WarningText
+import com.intersoft.utils.DateTimeManager
+
+@Composable
+fun EditEventPage(
+    eventId: Int,
+    eventName: String,
+    description: String,
+    selectedDateInMillis: Long,
+    startTimeInMillis: Long,
+    durationInMillis: Long,
+    maxNumberOfParticipants: Int,
+    location: String,
+    hostId: Int,
+    onEditEvent: () -> Unit,
+    onCancelEditEvent: () -> Unit
```

Comment:
I would just pass a single EventModel parameter here instead of all the fields seperately.

Reply:
EventModel is in :data:event, this would break the planed system architecture